BEP131.md

```diff
@@ -32,64 +32,69 @@ Meanwhile, Binance Smart Chain has 21 active validators and many inactive valida
 
 ### 5.1 Overview
 ![overview](./assets/bep-131/5.1_overview.png)
+
 In the current version, the validator address is actually selected by Binance Chain, and Binance Smart Chain will receive a cross-chain package from Binance Chain every day to know which validators can be miners.
```

Comment:
this line should be removed?

Reply:
No, because in some MarkDown interpreter, it will cause chaos if no line here.